client/src/components/Companies/Facts.js

```diff
@@ -0,0 +1,183 @@
+import InfoIcon from "@material-ui/icons/Info";
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import Typography from "@material-ui/core/Typography";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import React, {useEffect, useState} from "react";
+import {makeStyles} from "@material-ui/core/styles";
+import axios from "axios";
+
+const style = makeStyles((theme) => ({
+    downdrop: {
+        width: '90%',
+        boxShadow: 'none',
+        paddingLeft: '0px',
+        paddingRight: '0px'
+    },
+    
+    heading: {
+        fontFamily: 'DM Sans',
+        color: '#3D3E3F',
+        fontSize: '16px'
+    },
+    expandMenu: {
+        backgroundColor: '#F2F2F2'
+    },
+}));
+
+const companyFacts = [
+    {
+        CompanyID: 0,
+        Heading: [],
+        Summary: [],
+        ID: []
+    }
+]
+
+export default function Facts(props) {
+    const [factCitation, setFactCitation] = useState([]);
+    const [showCitation, setShowCitation] = useState(false);
+    const [fact, setFact] = React.useState(companyFacts);
+    const [state, setState] = useState([])
+
+    useEffect(() => {
+        axios.post(
```

Comment:
Don't just copy the old code.  Should use 'Get' method here.
The misuse of http method is one reason we have this optimization.